_commands/tcpdump.md

```diff
@@ -0,0 +1,101 @@
+---
+---
+
+tcpdump
+-------
+
+tcpdump is a tool that allows you to view a dump of traffic on a network. By default, tcpdump will let you view the dump on your local network.
```

Comment:
Also might be worth noting tools like [Wireshark](http://wireshark.com) that you can use to analyze tcpdumps.